provider/postgis/postgis_internal_test.go

```diff
@@ -20,28 +19,20 @@ const TESTENV = "RUN_POSTGIS_TESTS"
 
 var defaultEnvConfig map[string]interface{}
 
-func GetTestPort() int {
-	port, err := strconv.ParseInt(os.Getenv("PGPORT"), 10, 32)
-	if err != nil {
-		// Since this is happening at init time, have a sane default
-		fmt.Fprintf(os.Stderr, "err parsing PGPORT: '%v' using default port: 5433", err)
-		return 5433
-	}
-	return int(port)
-}
-
 func getConfigFromEnv() map[string]interface{} {
-	port := GetTestPort()
+	// safe to ignore err, as we always return a value that can be parsed to int
+	port, _ := strconv.Atoi(ttools.GetEnvDefault("PGPORT", "5432"))
```

Comment:
This is not true, as the USER could pass in a value that would not parse to an int.  I think it would be better to have a GetEnvDefaultInt function that takes an int default value and returns an int. The other thing we can do; is create a new type  (say EnvironmentVar) and attach a set of functions to it.
e.g.:
```go

type EnvironmentVar string

// AsString will return the value stored in the environment variable if set  otherwise the default
func (env EnvironmentVar) AsString(default string) string {
    val, ok := os.LookupEnv(string(env))
    if !ok {
      return default
    }
    return val
}

// AsInt will return the value stored in the EnvironmentVar as an int, otherwise a default value. If the value stored in the EnvironmentVar can not be cocersed into an int the default value will be returned along will the cocersion error.
func (env EnvironmentVar) AsInt(default int) (int,error) {
    val, ok := os.LookupEnv(string(env))
    if !ok {
      return default, nil
   }
     v, err := strconv.Atoi(val)
    if err != nill  {
       return default, err
    }
    return v, nil
}
```